exports.go

```diff
@@ -7,6 +7,7 @@ import "strconv"
 type ExportList []*papi.IsiExport
 type Export *papi.IsiExport
 
+// Gets a list of all exports on the cluster
 func (c *Client) GetIsiExports() (ExportList, error) {
 	exports, err := c.api.GetIsiExports()
 	if err != nil {
```

Comment:
I don't see what's wrong.  The closing bracket for the if statement is lined up with the if statement and the closing bracket of the function is in the first column.  Am I missing something?